# CaptureCameraImage.srv
sensor_msgs/Image image
---
bool success
string message
